lib/chingu/fpscounter.rb

```diff
@@ -19,56 +19,59 @@
 #
 #++
 
-
 module Chingu
   #
   # Calculates a fps and a tick-time for use in update-calls
   # register_tick() must be called every game loop iteration
   #
-	class FPSCounter
+  class FPSCounter
```

Comment:
I don't think it makes sense to formally deprecate this class because we want to keep old games running indefinitely, but maybe the comment should mention that it has been obsoleted by the built-in Gosu.fps function.